sqlglot/parser.py

```diff
@@ -1511,6 +1535,21 @@ def _parse_index(self) -> exp.Expression:
             columns=self._parse_expression(),
         )
 
+    def _parse_create_table_index(self):
+        unique = self._match_text_seq("UNIQUE")
```

Comment:
no need to do text because unique and primary are tokens

Reply:
I'm not seeing a primary token (just primary key) - should I create one or use text?